kedro_mlflow/pipeline/pipeline_ml.py

```diff
@@ -37,11 +37,34 @@ def __init__(
         tags: Union[str, Iterable[str]] = None,
         inference: Pipeline,
         input_name: str,
+        conda_env: Union[str, Path, Dict[str, Any]] = None,
+        model_name: Union[str, None] = "model",
     ):
+        """Store all necessary information for calling mlflow.log_model in the pipeline.
+
+        Args:
+            nodes (Iterable[Union[Node,): The `node`s of the training pipeline.
+            inference (Pipeline): A `Pipeline` object which will be stored in mlflow and use the output(s) of the training pipeline (namely, the model) to predict the outcome.
+            input_name (str): The name of the dataset in the catalog.yml which the model's user must provide for prediction (i.e. the data)
+            tags (Union[str, Iterable[str]], optional): Optional set of tags to be applied to all the pipeline nodes. Defaults to None.
+            conda_env (Union[str, Path, Dict[str, Any]], optional): The minimal conda environment the inference `Pipeline` for prediction.
+                It can be either :
+                - a path to a "requirements.txt": In this case
+                the packages are parsed and a conda env with
+                your current python_version and these dependencies is returned
+                - a path to an "environment.yml" : data is loaded and used as they are
+                - a Dict : used as the environment
+                - None: a base conda environment with your current python version and your project version at training time.
+            Defaults to None.
```

Comment:
- [ ] This should be indented with one more tab

Reply:
Well spotted. I have also refactored the docstring to make shorter lines length for readibility.